beta-src/src/components/ui/WDInfoPanelButton.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+import { useState } from "react";
+import WDButton from "./WDButton";
+
+interface WDInfoPanelButtonProps {
+  name: "Cancel" | "Draw" | "Pause";
```

Comment:
i think we should make this an enum that is exported from enums folder so that we can use it for other stuff in the game.

```
export enum GameVote {
  CANCEL = "cancel",
  DRAW = "draw",
  PAUSE = "pause"
}
```

i think GameVote might be a good name...check with bryan